packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -166,40 +191,60 @@ export const createRdsLambdaRole = (roleName: string, stack: Construct, secretEn
           `arn:aws:ssm:*:*:parameter${secretEntry.database}`,
           `arn:aws:ssm:*:*:parameter${secretEntry.port}`,
         ],
-      })
-    )
+      }),
+    );
   }
-  role.attachInlinePolicy(
-    new Policy(stack, 'CloudwatchLogsAccess', {
-      statements: policyStatements,
+
+  role.attachInlinePolicy(new Policy(stack, 'LogsAndParametersAccess', {
+    statements: policyStatements,
+    policyName: `${roleName}Policy`,
+  }));
+
+  role.addToPolicy(
+    new PolicyStatement({
+      effect: Effect.ALLOW,
+      resources: ['*'],
+      actions: [
+        'ec2:CreateNetworkInterface',
+        'ec2:DescribeNetworkInterfaces',
+        'ec2:DeleteNetworkInterface',
+      ],
```

Comment:
'ec2:DescribeNetworkInterfaces' permission is required on resources '*' for lambda deployment in VPC.
https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html#vpc-permissions